### ISSUE TITLE
Quantitative/Text States

### ISSUE THREAD

**Issue by clairecleveland** [paleobot/pbot-dev#251]
Currently, we allow quantitative states which allow users to type anything into the quantitative field (numerical or text).

To allow the customization we promised early on, we "just" need to tweak this. 

- Continue to provide quantitative fields but allow users to enter their own name to the field instead of just calling it quantity. Restrict to numerical data?
- Duplicate quantity field and call it text instead of quantity and allow users to enter text. No numerical restriction.

I have listed this as enhancement, but should be considered core feature for V2 to provide customization we promised.

**Comment by aazaff** [comment]
I agree this is a V2 matter the value of labeling all quantities as quantity has implications for how queries/searches work.
